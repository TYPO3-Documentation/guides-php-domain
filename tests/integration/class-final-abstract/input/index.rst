===================================
abstract final class causes warning
===================================

..  php:class:: TYPO3\CMS\Core\Test
    :abstract:
    :final:

    Lorem Ipsum Dolor!
