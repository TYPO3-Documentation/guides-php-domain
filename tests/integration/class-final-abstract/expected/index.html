<!-- content start -->
    <div class="section" id="abstract-final-class-causes-warning">
            <h1>abstract final class causes warning</h1>
            <dl class="php class">
    <dt class="sig sig-object php" id="typo3-cms-core-test">
        <span class="sig-name modifier"><span class="pre">abstract</span></span> <span class="sig-name modifier"><span class="pre">final</span></span> <em class="property"><span class="pre">class</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\</span></span>
<span class="sig-name descname"><span class="pre">Test</span></span> <a class="headerlink" href="#typo3-cms-core-test" title="Permalink to this definition">Â¶</a>
    </dt>
    <dd>

    <p>Lorem Ipsum Dolor!</p>

    </dd>
</dl>
    </div>
<!-- content end -->
